.gitignore

```diff
@@ -1,4 +1,7 @@
-/.cache/
+build/
+.cache/
```

Comment:
It is not how the original file sorted:

> *.eggs/
> Gemfile.lock
> .jekyll-metadata

And the official github python gitignore doesn't seem to follow any order.
https://github.com/github/gitignore/blob/master/Python.gitignore